lisa/tools/kdump.py

```diff
@@ -669,6 +649,42 @@ def config_resource_disk_dump_path(self, dump_path: str) -> None:
         If the system memory size is bigger than 1T, the default size of /var/crash
         may not be enough to store the dump file, need to change the dump path
         """
+
+        """
+        path option is not supported by default initrd. Regenerated initrd will not
+        boot properly in ARM64. So, skip the test if it is ARM64 and Mariner-2.0
+        """
+        if (
+            self.node.os.information.version.major == 2
```

Comment:
Please make sure the code logic is for Mariner only by check OS type.

Reply:
As this is the `CBLMariner` override-function from the base class, is it really necessary to check the OS type here?

Reply:
Thank you for pointing out. You are right, it doesn't need.

Reply:
One more question, should it be like `>= 2`? How about Mariner 3?

Reply:
It's only applicable for 2.0. Mariner-3 is good.
That library loading issue got fixed.